examples/eng_wasm_contracts/contract_with_eth_calls/src/lib.rs

```diff
@@ -52,5 +58,23 @@ impl ContractInterface for Contract {
         let key = "code";
         eprint!("{}", a);
         write_state!(key => &a);
+        let sum: u64 = 12;
```

Comment:
why 12? is this value special?
if not, better set it to 0 or something

Reply:
why 0 then?
funny how you say `better set it to 0 or something`, then I set it to **something**, and that something is 12. Seriously, I don't care the value, I just want it to be something different than null and 0. The tests are plagued with random combination of values, this is no different.

Just from the same testfile:
- https://github.com/enigmampc/enigma-core/blob/c467f5f83253965330d39f9500718220fb97fd6c/examples/eng_wasm_contracts/contract_with_eth_calls/src/lib.rs#L31
- https://github.com/enigmampc/enigma-core/blob/c467f5f83253965330d39f9500718220fb97fd6c/examples/eng_wasm_contracts/contract_with_eth_calls/src/lib.rs#L51

So the answer is that is a discrete value that I can easily verify at the other end.

Reply:
That's fair, i can understand that.